src/k-provider/ovp/index.js

```diff
@@ -1,10 +1,12 @@
 // @flow
 import OVPProvider from './provider';
+import RequestBuilder from '../../util/request-builder';
+import {RESPONSE_PROFILE_TYPE} from './request-params/base-entry-response-profile';
 
 declare var __VERSION__: string;
 declare var __NAME__: string;
 
 const NAME = __NAME__ + '-ovp';
 const VERSION = __VERSION__;
 
-export {OVPProvider as Provider, NAME, VERSION};
+export {OVPProvider as Provider, NAME, VERSION, RequestBuilder, RESPONSE_PROFILE_TYPE as ResponseProfileType};
```

Comment:
in second thought we can with no exporting the response profile type as this is not provider enum but server enum.
also, we might want to export all provider enums under Enum object or something like this. so I don't want to keep this was for backward compatibility.

Reply:
done